vendor/knative.dev/pkg/injection/sharedmain/main.go

```diff
@@ -226,6 +229,10 @@ func MainWithConfig(ctx context.Context, component string, cfg *rest.Config, cto
 	profilingServer := profiling.NewServer(profilingHandler)
 
 	CheckK8sClientMinimumVersionOrDie(ctx, logger)
+	// HACK: should go away when we move away from < 4.11 releases
+	if err := CheckMinimumKubeVersion(kubeclient.Get(ctx).Discovery(), "1.24.0"); err == nil {
+		os.Setenv("OCP_SECCOMP_PROFILE_WITHOUT_SCC", "true")
```

Comment:
@skonto is that our env-var? If so, I'd prefix it slightly different, to not confuse w/ actual OCP bits